aspnetcore/mvc/models/file-uploads.md

```diff
@@ -735,6 +735,10 @@ A connection error and a reset server connection probably indicates that the upl
 
 If the controller is accepting uploaded files using <xref:Microsoft.AspNetCore.Http.IFormFile> but the value is `null`, confirm that the HTML form is specifying an `enctype` value of `multipart/form-data`. If this attribute isn't set on the `<form>` element, the file upload doesn't occur and any bound <xref:Microsoft.AspNetCore.Http.IFormFile> arguments are `null`. Also confirm that the [upload naming in form data matches the app's naming](#match-name-attribute-value-to-parameter-name-of-post-method).
 
+### Stream was too long
+
+The examples in this topic rely upon <xref:System.IO.MemoryStream> to hold the uploaded file's content. The size limit of a `MemoryStream` is `int.MaxValue`. If the app's file upload scenario requires holding file content larger than `int.MaxValue`, use an alternative approach that doesn't rely upon a single `MemoryStream` for holding an uploaded file's content.
```

Comment:
That's a really large amount of content to be held in memory. We should say use an alternate approach if you know file sizes are larger than a much more reasonable value such as 50 MB or so.